### ISSUE TITLE
Pre-load crashing because it thinks Forge 35.1.4 is an earlier version than 35.1.10

### ISSUE THREAD

**Issue by tractorbeam0** [OreCruncher/DynamicSurroundings#641]
**Mod Version:**

4.0.3.2

**Forge Version:**

35.1.4

**Link to client log:**

https://pastebin.com/hyGmqMP0

**Link to crash log:**

https://pastebin.com/3df9tvA3

**Description:**

Obviously I have a later version then 35.1.10 but because of the extra 0 at the end it thinks that 35.1.4 is an earlier version.

**Comment by OreCruncher** [comment]
That's not how version numbers work.  35.1.4 is older than 35.1.10.  You don't strip the 0 at the end.  And it is Forge that enforces these rules based on meta in the mods TOML file.

**Comment by tractorbeam0** [comment]
Sorry for writing the issue aggressively, I was having problems with mod
compatability for about 2 hours and had to convert every single mod from
1.12.2 to 1.16.2 because of one mod that didn't like one mod as the 1.12.2
version and when I did that it complained that this wasn't the latest
version for this mod.

I'm just here because I had to go into the forge version history to get the
right version, even though I had recommended installed. I did try latest as
well.

Anyways, recent update fixed the issue, thank you for that. I was able to
update forge again and keep other mods working.

Your mod is awesome. I might as well tell you while I have your attention.

Gavator234
